README.md

```diff
@@ -60,4 +60,4 @@ Reanimated library is licensed under [The MIT License](LICENSE).
 This project is supported by amazing people from [Expo.io](https://expo.io) and [Software Mansion](https://swmansion.com)
 
 [![expo](https://avatars2.githubusercontent.com/u/12504344?v=3&s=100 'Expo.io')](https://expo.io)
-[![swm](https://avatars0.githubusercontent.com/u/56880679?v=3&s=100 'Software Mansion')](https://swmansion.com)
+[![swm](https://logo.swmansion.com/logo?color=white&variant=desktop&width=150&tag=react-native-reanimated-github 'Software Mansion')](https://swmansion.com)
```

Comment:
what about `height`? I see only `width` 🤔

Reply:
Height is calculated automatically based on `width`